language-support/scala/bindings-akka/src/test/scala/com/digitalasset/ledger/client/binding/retrying/CommandRetryFlowUT.scala

```diff
@@ -35,7 +35,7 @@ class CommandRetryFlowUT extends AsyncWordSpec with Matchers with AkkaTest {
       .map {
         case Ctx(
             context @ RetryInfo(_, _, _, status),
-            SubmitRequest(Some(Commands(_, _, _, commandId, _, let, _, _, _)), tc)) =>
+            SubmitRequest(Some(Commands(_, _, _, commandId, _, let, _, _, _, _)), tc)) =>
```

Comment:
Since we're ignoring so many fields, we can probably just bind `Commands` to a variable and access it in the following block.